test/run-tests.py

```diff
@@ -189,7 +199,8 @@ def Get(self):
     return self.value[0]
 
 
-def RunCommandWithTimeout(command, cwd, timeout, console_out=False, env=None):
+def RunCommandListWithTimeout(command, cwd, timeout, console_out=False,
```

Comment:
Why rename this?  It still takes a single command doesn't it?

Reply:
Yeah, good point. I wanted to differentiate between passing a `Command` object and a list of strings (`CommandList`) but maybe it's not worth it.